mailbag/derivatives/pdf.py

```diff
@@ -0,0 +1,43 @@
+import os
+from subprocess import Popen, PIPE
+from mailbag.derivative import Derivative
+
+
+class ExampleDerivative(Derivative):
+    derivative_name = 'pdf'
+
+    def __init__(self, email_account, **kwargs):
+        print("Setup account")
+        super()
+
+    def do_task_per_account(self):
+        print(self.account.account_data())
+
+    def do_task_per_message(self, message, args):
+        wkhtmltopdf = 'wkhtmltopdf.exe'
+        html = message.HTML_Body
+        From_name=message.From.split("<")
+        To_name=message.To.split("<")
+        #htmlnew="<table>  <tr>    <th>From</th>    <th>To</th> </tr>"  + "<tr> <td>"+a+d[1][:-1]+"</td>"+"<td>"+b+c[1][:-1]+"</td>  </tr>"+"</table>"
```

Comment:
You're also going to need much more than just to and from headers. We I would use all of the string fields in the model, except for the bodies and error. There also should be a way to used the model attribute name as the field instead of manually writing "From," "To," etc.